src/main/java/org/jabref/gui/preferences/EntryEditorTabViewModel.java

```diff
@@ -106,7 +109,13 @@ public void storeSettings() {
         } else if (firstNameModeFullProperty.getValue()) {
             firstNameMode = AutoCompleteFirstNameMode.ONLY_FULL;
         }
-
+        if(initialAutoCompletePreferences.shouldAutoComplete()!=enableAutoCompleteProperty.getValue()){
```

Comment:
you have a lot of checkstyle issues here. Please setup the JabRef Code Style and make your life easier

Reply:
Thanks for your review.